cit3.0-web/src/components/OpportunityView/Resource.js

```diff
@@ -0,0 +1,134 @@
+import PropTypes from "prop-types";
+import { Row, Col } from "react-bootstrap";
+import { v4 } from "uuid";
+import NumberFormat from "react-number-format";
+import "./Resource.css";
+
+/**
+ * @param {Object} resources from redux state: opportunity
+ */
+function displayResources(resources) {
+  // toDisplay to control nesting of elements
+  const toDisplay = {};
+
+  // Process resource entries
+  Object.entries(resources).forEach((resource) => {
+    // Datapoint element
+    let element = null;
+    let multiValues = [];
+
+    // Insert Category section/datapoint
+    if (!toDisplay[resource[1].title]) {
+      toDisplay[resource[1].title] = {};
+    }
+
+    // Individual data points
+    if (
+      resource[1].subtitle &&
+      typeof toDisplay[resource[1].title][resource[1].subtitle] === "undefined"
+    ) {
+      toDisplay[resource[1].title][resource[1].subtitle] = {};
+    }
+
+    // Different markup for datapoint types
+    /* eslint-disable no-fallthrough */
```

Comment:
Are we able to remove this disable line? I looked it up and this code appears to be following the rules correctly. Was it still causing errors for some reason?

Reply:
Removed, as a previous solution needed it